bitnami/kafka/docker-compose-cluster.yml

```diff
@@ -5,36 +5,54 @@ version: "2"
 
 services:
   kafka-0:
-    image: docker.io/bitnami/kafka:3.5
+    image: docker.io/bitnami/kafka:3.5.1
```

Comment:
This should **not** be updated. `bitnami/kafka:3.5` is a rolling tag, and indeed points to `bitnami/kafka:3.5.1` as of now. It will also point to `3.5.2` whenever this version is released, so there is no need to update this line every time a new version is reelased.

Reply:
I got it.